src/app/components/core/notifications/add-notification-wizard/add-notification-wizard.component.ts

```diff
@@ -215,15 +223,11 @@ export class AddNotificationWizardComponent implements OnInit, OnDestroy {
     const previousButton = <HTMLButtonElement>document.getElementById('previous');
     switch (+id) {
       case 1:
-        if (formValues['name'].trim() === '') {
-          this.isValidName = false;
-          return;
-        }
         nxtButton.textContent = 'Next';
         previousButton.textContent = 'Previous';
         if (formValues['name'].trim() !== '') {
           this.payload.name = formValues['name'];
-          this.payload.description = this.description.nativeElement.value;
+          this.payload.description = formValues['description'];
```

Comment:
remove extra newline after = ?

Reply:
where?

Reply:
This is good in actual; while reviewing, in browser, I'm seeing

<img width="683" alt="Screenshot 2023-02-01 at 8 49 05 PM" src="https://user-images.githubusercontent.com/1974003/216083769-f3216ea2-b96b-4d87-b957-45dbb7360831.png">